docs/css.md

```diff
@@ -6,10 +6,12 @@ Webpacker supports importing CSS, Sass and SCSS files directly into your JavaScr
 
 ## Import styles into your JS app
 
+Stylesheets end with '.modules.*' is treated as [CSS Modules](https://github.com/css-modules/css-modules).
```

Comment:
Could we please make a new documentation for this? We still support regular sass or css imports without css modules.